deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py

```diff
@@ -247,3 +251,47 @@ def check_vulnerability_scan_remove_alerts(wazuh_alerts_monitor, package, cve):
                                                               f"to a package removal/update or a system upgrade",
                               prefix='.*', error_message='Could not find the alert which says that the package '
                                                          f"{package} is no longer vulnerable with {cve}")
+
+
+def check_provider_database_update_start_log(provider_name):
+    """Check the provider database update start event in ossec.log
+
+    Args:
+        provider_name (str): Provider name of the downloaded feed.
+    """
+    check_vuln_detector_event(timeout=vd.VULN_DETECTOR_FAST_TIMEOUT,
+                              callback=f"Starting '{provider_name}' database update",
+                              error_message=f"Could not find {provider_name} update starting log")
+
+
+def check_provider_database_update_finish_log(provider_name):
+    """Check the provider database update finish event in ossec.log
+
+    Args:
+        provider_name (str): Provider name of the downloaded feed.
+    """
+    check_vuln_detector_event(timeout=vd.VULN_DETECTOR_GLOBAL_TIMEOUT,
+                              callback=f"The update of the '{provider_name}' feed finished",
+                              error_message=f"Could not find {provider_name} feed finished log")
+
+
+def check_fetching_feed_log(feed_path):
+    """Check the fetching feed file event in ossec.log
+
+    Args:
+        feed_filename (str): filename to be fetched.
+    """
+    check_vuln_detector_event(timeout=vd.VULN_DETECTOR_GLOBAL_TIMEOUT,
+                              callback=rf"Fetching .* from '{feed_path}'",
+                              error_message=f"Could not find {feed_path} feed fetching log")
+
+
+def check_refresh_feed_log(provider_name):
+    """Check the inserting feed event in ossec.log
+
+    Args:
+        feed_filename (str): filename to be fetched.
+    """
+    check_vuln_detector_event(timeout=vd.VULN_DETECTOR_GLOBAL_TIMEOUT,
+                              callback=f"Refresh of '{provider_name}' database finished",
+                              error_message=f"Could not find {provider_name} feed refresh log")
```

Comment:
Add a new line at the end according to PEP-8

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2538/commits/762b15217466b07a5e4bfa23bcf2c647e1848760